### PR TITLE
Update maintenance incidents table

### PR DESCRIPTION
**Pull request by bgeuken**
* Merge release targets and build result column
* Make release target project linkable
* Add request creation time next to request flags

Before:

https://build.opensuse.org/projects/openSUSE:Maintenance/maintenance_incidents

After:

### CODE REVIEW

**Comment by bgeuken** [comment]
[Maintenance incidents](https://trello.com/c/4hbIZuK9/1095-maintenance-incidents)

**Comment by vpereira** [comment]
@bgeuken the categories could be just recommended/security. I'm not sure why are we adding the "missing patchinfo" as category?

**Comment by bgeuken** [comment]
I guess it's to raise that a patchinfo has to be added. It's also linked, so one can just click on it to generate one.

So, I don't know. But since it was already there and it seems useful I kept it.

File: src/api/app/datatables/maintenance_incident_datatable.rb [hunk 1]
```diff
@@ -26,8 +26,7 @@ def data
         category: category_cell(record, patchinfo),
```
Comment (vpereira) [inline src/api/app/datatables/maintenance_incident_datatable.rb]: Do it make sense to have in category, something that isn't a category?

Comment (bgeuken) [reply src/api/app/datatables/maintenance_incident_datatable.rb]: So far it made sense for the maintenance team (bento UI). I am reluctant to change this without talking with them first.

Note that this is already part of the view, old and new UI.

Comment (vpereira) [reply src/api/app/datatables/maintenance_incident_datatable.rb]: ah ok, sorry for the noise

**Pull request merged**